### ISSUE TITLE
Ray Dashboard Head-node CLI [autoscaler]

### ISSUE THREAD

**Issue by mccrearyd** [ray-project/ray#8450]
The ray dashboard is nice for the browser, however I find it kind of a pain to deal with opening the browser every time I want to view the progress, also it doesn't work very well without ssh tunneling.

I am a huge fan of `nvidia-smi` and `top` for monitoring a single machine, works really well with `tmux`. I think theres a lot of potential for ray to have something similar for cluster resource viewing, especially with their agent logical views and such. 

![Screenshot from 2020-05-14 16-55-12](https://user-images.githubusercontent.com/26421036/81997122-b7be6d00-9603-11ea-85bc-e08496f32178.png)

Basically just take all of the information being provided to the ray dashboard web page, and allow access to the headnode through the cli, something like `raysources`, and live monitoring with `watch raysources`, much like `watch nvidia-smi`.

**Comment by rkooo567** [comment]
Hi, @McCrearyD! Thanks for the feature request. I do think it is a good idea to have a CLI dashboard, but we are not sure when we will be able to have bandwidth to work on this. We will definitely add a task to the backlog!

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
